extensions/interactions/protractor.js

```diff
@@ -50,7 +50,7 @@ var INTERACTIONS = {
   GraphInput: require('./GraphInput/protractor.js'),
   ItemSelectionInput: require('./ItemSelectionInput/protractor.js'),
   LogicProof: require('./LogicProof/protractor.js'),
-  MathExpressionInput: require('./MathExpressionInput/protractor.js'),
```

Comment:
It doesn't look like you've deleted the actual `MathExpressionInput` folder, so why is it no longer included in this file?

Reply:
I've added a comment explaining this in the description now, PTAL, and let me know if there are still any ambiguities.

Reply:
This makes sense!